src/stories/lift-component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+
+import Lift from '../components/schedule-visualizer/lift';
+
+export interface LiftComponentProps {
+    currentFloor: string;
+    lift: RomiCore.Lift;
+    liftState: RomiCore.LiftState;
+    renderInfoPanel(): JSX.Element;
+    currentMode(): JSX.Element;
+    motionState(): JSX.Element;
```

Comment:
Are these type JSX.Element or React.ReactElement?

This is an interesting link:
https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

Reply:
Should we also prefix these with `render*` to make things easier to understand?

Reply:
The functions are JSX.Element since they have `any` for their props and type